apps/sms/views/shared/js/time_headers.js

```diff
@@ -43,23 +43,18 @@
         return;
```

Comment:
thanks for making me look at this file again... I think there are a lot of things to eventually change here :)